app/src/main/res/layout/fragment_share.xml

```diff
@@ -54,6 +54,14 @@
             android:background="?borderPrimary"
             app:layout_constraintBottom_toTopOf="@id/appsShareLayout" />
 
+        <androidx.compose.ui.platform.ComposeView
+            android:visibility="gone"
+            android:id="@+id/save_pdf"
+            android:layout_width="match_parent"
+            android:layout_height="72dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/appsShareLayout" />
+
```

Comment:
Or perhaps a divider in the XML that has conditional visibility like the ComposeView?